src/main/java/com/github/dcysteine/neicustomdiagram/generators/gregtech5/oreprocessing/LayoutHandler.java

```diff
@@ -113,8 +126,23 @@ private Layout buildRawOreMacerateLayout() {
                 .putSlotGroup(SlotGroupKeys.RAW_ORE_MACERATE, outputSlots).build();
     }
 
+    private Layout buildTrueRawOreMacerateLayout() {
+        Lines lines = Lines.builder(Grid.GRID.grid(6, 2)).addArrow(Grid.GRID.edge(6, 7, Grid.Direction.N)).build();
```

Comment:
I think it might be good to add a line segment from `(4, 2)` to `(6, 2)` here. Currently, it looks like we only get that line from the ore wash layout, so if there is no ore wash recipe, then the raw ore macerate slots will end up disconnected.

Reply:
On it